services/inspection-report/src/views/App/index.js

```diff
@@ -58,11 +59,12 @@ export default function App() {
 
   if (hasToken) {
     return (
-      <div>
+      <div style={{ minHeight: '100vh', display: 'flex', flexDirection: 'column' }}>
```

Comment:
Shouldn't it be `<View />` in `services/inspection-report/src/components/View/index.js` ?